test/test.cpp

```diff
@@ -10,7 +10,7 @@
 
 #include "timsort.hpp"
 
-#if ENABLE_STD_MOVE
+#if GFX_TIMSORT_USE_CXX11
```

Comment:
The `ENABLE_STD_MOVE` macro was actually removed by the e39ca95ee2ca4e717fce73e5a0088c82944d14dd change in 2017, so this `#warning` was erroneously always printing "disabled"

I think the cleanest thing to do is to just let the `GFX_TIMSORT_USE_CXX11` define leak out of timsort.hpp (i.e. it's not `#undef`ed like the other macros) so it can be tested here.